app/helpers/audit_helper.rb

```diff
@@ -59,6 +59,6 @@ def activity(type_of_activity, args = nil)
 
   def define_api_user(audit)
     change_requests = %w[description_change_request_received red_line_boundary_change_request_received document_change_request_received document_create_request_received]
-    change_requests.include?(audit.activity_type) ? "Applicant / Agemt" : audit.api_user.name
+    change_requests.include?(audit.activity_type) ? "Applicant / Agent via #{audit.api_user.name}" : audit.api_user.name
```

Comment:
I think we want to say "Applicant / Agent via #{audit.api_user.name}" for all api actions (I mean the only other one is the creation of an application, which is also the applicant via the API).

Reply:
Unfortunately at the moment we don't submit the api_user from `bops-applicants` so it's a very temporary fix until we can deploy the permanent fix to `bops_applicants`